### PR TITLE
Bug/addressed feedback

### PR DESCRIPTION
**Pull request by angelnikolov**
Addressed all the feedback here:
https://www.google.com/url?q=https://art19.com/shows/gladiator-aaron-hernandez-and-football-inc/episodes/59d91c5d-824d-45ae-98d9-6ae13d577432&sa=D&ust=1546876204337000&usg=AFQjCNFkhboewDSQBWKub6iJiTco_AuJmQ

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -66,12 +65,12 @@ Fliplet().then(() => {
       Fliplet.Widget.toggleCancelButton(true);
       Fliplet.Widget.toggleSaveButton(true);
       media.selectedFiles = providerData.data.length === 1 ? providerData.data[0] : providerData.data;
-
       providerInstance = null;
       $('.audio .add-audio').text('Replace audio');
       $('.audio .info-holder').removeClass('hidden');
       $('.audio .file-title span').text(media.selectedFiles.name);
       Fliplet.Widget.autosize();
+      await save(false);
```
Comment (squallstar) [inline js/interface.js:74]: wouldn't work on IE 11 I'm afraid

File: js/interface.js [hunk 2]
```diff
@@ -109,14 +108,25 @@ Fliplet().then(() => {
...
       audioByUrl.removeClass('show');
+      
     } else {
       audioByFilePicker.removeClass('show');
       audioByUrl.addClass('show');
     }
+
+    widgetInstanceData.audioType = $(this).val();
+
+    await Fliplet.Widget.save(widgetInstanceData);
```
Comment (squallstar) [inline js/interface.js:137]: rewrite as a promise :-)

Comment (angelnikolov) [reply js/interface.js:137]: Webpack will do this for me maaan :) We transplin'

**Review by squallstar — APPROVED** [review]

File: js/interface.js (hunk 2 above)
Comment (squallstar) [reply js/interface.js:137]: ahhhh forgot about that :-)

nice one

**Review by tonytlwu — APPROVED** [review]

**Pull request merged**